### PR TITLE
/specifications/ updated with new specification version - css fix - DeviceTree Releases pulled from Github using their API - DeviceTree Members added to footer

### PR DESCRIPTION
**Pull request by kylekirkby**
I've updated the /specifications/ page to link to the new version not sure what to change the text beneath to as this will also need updating. Added a quick fix for the Sub Nav colour issue - I will make sure this gets addressed in the latest version of the Jumbo Jekyll Theme.

@shovanuk @glikely

### CODE REVIEW

File: specifications/README.md [hunk 1]
```diff
@@ -6,7 +6,10 @@ keywords: Devicetree, Specification, release, v1.0
 ---
 ## **The Devicetree Specification**
 
-<a href="/downloads/devicetree-specification-v0.1-20160524.pdf" target="_blank"><strong>Device Tree Specification Release 0.1</strong></a><br>
```
Comment (robherring) [inline specifications/README.md]: We probably want to keep the prior versions.

File: specifications/README.md [hunk 2]
```diff
@@ -6,7 +6,10 @@ keywords: Devicetree, Specification, release, v1.0
...
+<a href="/downloads/devicetree-specification-v0.2.pdf" target="\_blank">
+<strong>Device Tree Specification Release 0.2</strong>
+</a>
+<br>
 
 This is the first formal baseline release of the device tree specification. This document is a first iteration of ePAPR 1.1 and picks up where it left off in 2012.
```
Comment (robherring) [inline specifications/README.md]: This applies only to 0.1.

File: specifications/README.md [hunk 3]
```diff
@@ -6,7 +6,10 @@ keywords: Devicetree, Specification, release, v1.0
...
 The next release v0.2 is due to be tagged before the end of December 2017.
```
Comment (robherring) [inline specifications/README.md]: This needs to be removed.

File: specifications/README.md (hunk 1 above)
Comment (kylekirkby) [reply specifications/README.md]: How do you want it listed on the page ? Just beneath 0.2 ?

Comment (robherring) [reply specifications/README.md]: Yes. Perhaps have "Current Release" and "Past Releases" sections.

The other option would be to have separate pages for current and past.

**Comment by shovanuk** [comment]
@robherring if you are happy please merdge the PR.

**Comment by robherring** [comment]
LGTM. I'll wait on merging for @glikely to comment. He's out til Mon.

**Comment by kylekirkby** [comment]
Hi Guys can this be merged ? @glikely @shovanuk @robherring

**Comment by glikely** [comment]
I haven't tested it, but it looks good to me.

**Comment by kylekirkby** [comment]
It looks like this when built - 
/specifications/
fetched_from_api

/releases/
github_repo_releases

**Comment by kvekaria** [comment]
👍

**Comment by glikely** [comment]
Ah, nice. I like how /releases it auto-populated. Can you make the /specifications page be autogenerated also?

**Pull request merged**